client/src/features/modal/Modal.jsx

```diff
@@ -2,7 +2,7 @@ import { AnimatePresence } from "framer-motion";
 import Backdrop from "./Backdrop";
 import { motion } from "framer-motion";
 
```

Comment:
Optionally you could add a check for the `onBackdropClick` type with `prop-types`. That way you get an error on page load of an invalid `onBackdropClick` prop is passed in.
```suggestion
import PropTypes from "prop-types";

// Modal component definition

Modal.propTypes = {
  onBackdropClick: PropTypes.func,
};
```

Like this
![image](https://github.com/Together-100Devs/Together/assets/19908880/6546e0fb-42f5-40c2-9c2e-be37b8325dca)

Reply:
If a redesign for the event modal is in the works like @intelagense mentioned then this may not be that important. Just my two cents.

Reply:
> If a redesign for the event modal is in the works like @intelagense mentioned then this may not be that important. Just my two cents.

I'm not sure the redesign will be coming anytime soon.